content/kubernetes-backing-services/monitoring/kube-prometheus.md

```diff
@@ -0,0 +1,134 @@
+* [Exporter kube dns](https://github.com/coreos/prometheus-operator/tree/master/helm/exporter-kube-dns)
+* [Exporter kube etcd](https://github.com/coreos/prometheus-operator/tree/master/helm/exporter-kube-etcd)
+* [Exporter kube scheduler](https://github.com/coreos/prometheus-operator/tree/master/helm/exporter-kube-scheduler)
+* [Exporter kube state](https://github.com/coreos/prometheus-operator/tree/master/helm/exporter-kube-state)
+* [Exporter kubelets](https://github.com/coreos/prometheus-operator/tree/master/helm/exporter-kubelets)
+* [Exporter kubernetes](https://github.com/coreos/prometheus-operator/tree/master/helm/exporter-kubernetes)
+* [Exporter node](https://github.com/coreos/prometheus-operator/tree/master/helm/exporter-node)
+* [Grafana](https://github.com/coreos/prometheus-operator/tree/master/helm/grafana)
+
+# Dependencies
+
+[Prometheus Operator]({{< relref "kubernetes-backing-services/monitoring/prometheus-operator.md" >}})
+
+# Install
+
+You can install `kube-prometheus` in a few different ways, but we recomend to use the [Master Helmfile](https://github.com/cloudposse/geodesic/blob/master/rootfs/conf/kops/helmfile.yaml).
+
+## Install with Master Helmfile
+
+To install `kube-prometheus` run:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Install kube-prometheus" %}}
+```
+helmfile -f /conf/kops/helmfile.yaml --selector namespace=monitoring,chart=kube-prometheus sync
+```
+{{% /dialog %}}
+
+These are some of the environment variables you may want to configure:
+
+* `KUBE_PROMETHEUS_REPLICA_COUNT` -Count of `prometheus` pods
+* `KUBE_PROMETHEUS_IMAGE_TAG` -Version of [`prometheus` image](https://quay.io/repository/prometheus/prometheus)
+* `KUBE_PROMETHEUS_ALERT_MANAGER_REPLICA_COUNT` - Count of `alertmanager` pods
+* `KUBE_PROMETHEUS_ALERT_MANAGER_IMAGE_TAG` - Version of [`alert manager` image](https://quay.io/repository/prometheus/alertmanager)
+* `KUBE_PROMETHEUS_ALERT_MANAGER_SLACK_WEBHOOK_URL` - [Slack webhook url](https://api.slack.com/incoming-webhooks) that would be triggered on alert
+* `KUBE_PROMETHEUS_ALERT_MANAGER_SLACK_CHANNEL` - [Slack channel](https://get.slack.help/hc/en-us/articles/201402297-Create-a-channel) where alerts would be posted
+
+Environment variables can be specified in the Geodesic Module's `Dockerfile` or using [Chamber]({{< relref "tools/chamber.md" >}}) storage, which is recommended for all secrets.
+
+## Install with Custom Helmfile
+
+Add to your [Kubernetes Backing Services](/kubernetes-backing-services) Helmfile this code
+
+{{% include-code-block  title="helmfile.yaml" file="kubernetes-backing-services/monitoring/examples/prometheus-operator-helmfile.yaml" language="yaml" %}}
+
+Then follow the instructions for running [`helmfile sync`]({{< relref "tools/helmfile.md" >}}).
+
+# Extending
+
+You can extend default functionality of `kube-prometheus` making it collect custom
```

Comment:
You can extend the default functionality of `kube-prometheus` to collect custom
metrics and send alerts based on the metrics, and display the metric in Grafana charts